dev-vcs/tortoisehg/tortoisehg-5.3.1.ebuild

```diff
@@ -2,14 +2,14 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-PYTHON_COMPAT=( python2_7 )
+PYTHON_COMPAT=( python3_6 )
 
 inherit desktop distutils-r1
 
 if [[ ${PV} != *9999* ]]; then
 	KEYWORDS="~amd64 ~x86"
 	SRC_URI="https://www.bitbucket.org/${PN}/targz/downloads/${P}.tar.gz"
-	HG_DEPEND=">=dev-vcs/mercurial-5.0 <dev-vcs/mercurial-5.1"
+	HG_DEPEND="~dev-vcs/mercurial-$PV"
```

Comment:
Always use braces, i.e. `${PV}`. Also, why do you need to force a single specific version? This is going to be a maintenance nightmare.

Reply:
Ok, sure. I've thought that It depends from hard certain versions but it don't